### ISSUE TITLE
[Heads-up] Upcoming WebAssembly Debugging changes in CDP

### ISSUE THREAD

**Issue by bmeurer** [microsoft/vscode-chrome-debug-core#547]
Following up on https://github.com/Microsoft/vscode-chrome-debug-core/issues/193, I wanted to give everyone here a heads up that we are in the process of changing the way WebAssembly is exposed via the CDP (Chromium DevTools Protocol). Previously we had exposed individual functions as so-called *fake scripts* via the protocol, in a way that the engine gives you a text format. In the future we'll only expose one script per Wasm module, and will not provide the text format anymore, but only the raw Wasm wire bytes.

The Chromium DevTools front-end is going to turn that into text format instead, probably leveraging an improved version of https://github.com/wasdk/wasmparser (which is also used in Firefox DevTools).

Part of this is that source locations will be consistently represented in terms of Wasm wire byte offsets. http://bit.ly/devtools-wasm-interface contains background information around these changes.

cc @rreverser @hashseed

**Comment by roblourens** [comment]
Thanks for the heads up! cc @connor4312 

Is there an issue we can watch to track progress in v8 or the devtools frontend?

**Comment by bmeurer** [comment]
Yes, the tracking issue is [chromium:1013527](https://bugs.chromium.org/p/chromium/issues/detail?id=1013527) as referenced from the http://bit.ly/devtools-wasm-interface document.
